letsencrypt/scripts/main.py

```diff
@@ -102,17 +114,18 @@ def main():  # pylint: disable=too-many-branches
     logger.setLevel(logging.INFO)
     if args.use_curses:
         logger.addHandler(log.DialogHandler())
-        displayer = display.NcursesDisplay()
+        displayer = display_util.NcursesDisplay()
     else:
-        displayer = display.FileDisplay(sys.stdout)
+        displayer = display_util.FileDisplay(sys.stdout)
+
     zope.component.provideUtility(displayer)
 
     if args.view_config_changes:
         client.view_config_changes(config)
         sys.exit()
 
-    if args.revoke:
-        client.revoke(config)
+    if args.revoke or args.rev_cert or args.rev_key:
```

Comment:
`args.rev_cert and args.rev_key` will always evaluate to true, since `read_file` always returns a two-element tuple. Unless... you meant `args.rev_cert is not None or args.rev_key is not None`, where `None` comes as a default argument parameter?